app/src/main/java/com/example/fittracker/ConstantUtils.java

```diff
@@ -19,4 +19,14 @@
     public static final String DATAHANDLER_TYPE = "text/plain";
     public static final Character COMA = ',';
     public static final String MAIL_HOST = "smtp.gmail.com";
+    public static final String WEATHER_BASE_URL = "https://api.openweathermap.org";
+    public static final int LOCATION_REQUEST_CODE = 1;
+    public static final String MARVEL_API_KEY_PUBLIC = "08cd7f2cd10462f4d20ab91d76ecec33";
+    public static final String MARVEL_API_KEY_PRIVATE = "82460263f65b53b64520b5a4fc43754a77cc1d85";
+    public static final String WEATHER_API_KEY = "852c6dcc0bd878109bd0fc7bb1566c3e";
+    public static final String WEATHER_UNITS = "metric";
+    public static final String WEATHER_ICON_URL = "https://openweathermap.org/img/w/";
```

Comment:
put this in a global variable in gradle pls

Reply:
Done